tests/testthat/helper-functions.R

```diff
@@ -38,3 +38,24 @@ dt_copies <- function(...) {
 dt_compare_all <- function(ref_copies, ...) {
   all(mapply(function(l, r) all(l == r), ref_copies, list(...)))
 }
+
```

Comment:
this being used now? looks like maybe vestigal.

Reply:
idk - seems like another issue.

Reply:
not super clear from the prompt, but this refers to new code introduced in this PR (`check_r_version`), but that isn't apparently used anywhere. (not newly vestigal - vestigal w/in this PR)

Reply:
This is being used once in the testing.